### PR TITLE
Set log level feature

### PR DESCRIPTION
**Pull request by yyyogev**

### CODE REVIEW

File: src/main/java/io/logz/jmx2logzio/Utils/MetricsPipeline.java [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 public class MetricsPipeline {
     private static final DateTimeFormatter timestampFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSZ").withZone(ZoneId.of("UTC"));
-    private static final Logger logger = LoggerFactory.getLogger(MetricsPipeline.class);
+    private final Logger logger = (Logger) LoggerFactory.getLogger(MetricsPipeline.class);
```
Comment (idohalevi) [inline src/main/java/io/logz/jmx2logzio/Utils/MetricsPipeline.java]: Why you deleted `static`?

File: src/main/java/io/logz/jmx2logzio/clients/JavaAgentClient.java [hunk 2]
```diff
@@ -30,7 +29,7 @@
 
 public class JavaAgentClient extends MBeanClient {
 
-    private static final Logger logger = LoggerFactory.getLogger(JavaAgentClient.class);
+    private final Logger logger = (Logger) LoggerFactory.getLogger(JavaAgentClient.class);
```
Comment (idohalevi) [inline src/main/java/io/logz/jmx2logzio/clients/JavaAgentClient.java]: same as above - static

File: src/main/java/io/logz/jmx2logzio/Jmx2LogzioJavaAgent.java [hunk 3]
```diff
@@ -28,18 +30,31 @@
...
     private static final int INDEX_OF_KEY = 0;
     private static final int INDEX_OF_VALUE = 1;
+    private static final String LOG_LEVEL_DEBUG = "DEBUG";
 
     public static void premain(String agentArgument, Instrumentation instrument) {
 
         logger.info("Loading with agentArgument: {}", agentArgument);
         Config finalConfig = getIntegratedConfiguration(agentArgument);
-        Jmx2LogzioConfiguration jmx2LogzioConfiguration = new Jmx2LogzioConfiguration(finalConfig);
+        Level logLevel = Level.WARN;
```
Comment (idohalevi) [inline src/main/java/io/logz/jmx2logzio/Jmx2LogzioJavaAgent.java]: If in the xml it's already WARN why do you need to set warn as a default here?

File: src/test/java/io/logz/jmx2logzio/LogzioSenderTest.java [hunk 4]
```diff
@@ -28,15 +28,15 @@
...
+    private final Logger logger = (Logger) LoggerFactory.getLogger(LogzioSenderTest.class);
     private Jmx2LogzioConfiguration config;
     private ClientAndServer mockServer;
     private HttpRequest[] recordedRequests;
     private MockServerClient mockServerClient = null;
 
     @BeforeTest
     private void startMockServer() {
-        logger.info("starting mock server");
+logger.info("starting mock server");
```
Comment (idohalevi) [inline src/test/java/io/logz/jmx2logzio/LogzioSenderTest.java]: info -> debug

File: src/main/java/io/logz/jmx2logzio/Utils/MetricsPipeline.java (hunk 1 above)
Comment (yyyogev) [reply src/main/java/io/logz/jmx2logzio/Utils/MetricsPipeline.java]: no reason for it to be static

File: src/main/java/io/logz/jmx2logzio/Jmx2LogzioJavaAgent.java (hunk 3 above)
Comment (yyyogev) [reply src/main/java/io/logz/jmx2logzio/Jmx2LogzioJavaAgent.java]: the xml property refers to java.utils.logging.Logger and not to logback's logger

**Pull request merged**